reference/orders.v3.yml

```diff
@@ -1924,13 +1928,16 @@ components:
               type: object
               description: |
                 A store credit model.
+              nullable: true
               properties:
                 remaining_balance:
                   description: |
                     Remaining balance of shopperʼs store credit.
                   type: number
                   format: float
                   example: 35.42
+            custom_provider_field_result: 
```

Comment:
If someone could provide the schema and description for this, that would be great

Reply:
@bigcommerce/team-payments 🙏

Reply:
Here you go with the schema details.
```
custom_provider_field_result:
  type: object
  description: custom payment provider field result model.
  title: Custom Payment Provider Field
  nullable: true
  properties:
    receipt_number:
      type: string
      description: |
        The receipt number associated with the transaction.
      nullable: true
    authorization_code:
      type: string
      description: |
        Authorization code for the transaction.
      nullable: true
    fraud_response:
      type: string
      description: |
        The fraud response associated with the transaction.
      nullable: true
    amount_received:
      type: number
      format: float
      description: |
        The amount received for the transaction, divided by 100 to convert to the correct currency format.
      nullable: true
```